Changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Language Features:
  * Allow to obtain the address of a linked library with ``address(LibraryName)``.
+ * Allow to obtain the selector of public or external library functions via a member ``.selector``.
```

Comment:
Do you want to get this into the release?

Reply:
I don't mind either way.

Reply:
I would say, it doesn't make sense to block the release for it.

Reply:
I'll move it once the new version entry is on ``develop``.